rpcs3/Emu/RSX/VK/VKFormats.cpp

```diff
@@ -17,12 +17,24 @@ namespace vk
 		{
 			if (support.d24_unorm_s8) return VK_FORMAT_D24_UNORM_S8_UINT;
 			if (support.d32_sfloat_s8) return VK_FORMAT_D32_SFLOAT_S8_UINT;
+#ifdef __APPLE__
+			// Allow Intel and NVIDIA GPUs on certain Macs to run at least some games
+			rsx_log.error("No hardware support for z24s8. Graphics may not render correctly.");
+			return VK_FORMAT_B8G8R8A8_UNORM;
```

Comment:
No, you need to return a usable depth format, returning a color format is guaranteed to crash unless you're running triangle test only.
D32_SFLOAT_s8_UINT is the safer of the two.